### ISSUE TITLE
Underwater-quest bug

### ISSUE THREAD

**Issue by LordHelmchen98** [Wynncraft/Issues#6083]
So recently I was travelling around with a friend who had to do the "Underwater"-quest. As I went with my lvl 100 Mage I was curious if I was able to kill the new boss (Ithink its a chest) from outside through the wall. So I tried it and saw it is possible. After I did that I realized the drop spawns on the bottom of the ocean above. I guess thats not how it is supposed to work.
I recorded it and uploaded it to youtube here: [https://youtu.be/806NxW586M8](url)

**Comment by Moe-Ronickah** [comment]
@AckroWynn

**Comment by LordHelmchen98** [comment]
So hmm this is still not fixed. Did you miss this thread or is that hard to fix? @AckroWynn

**Comment by Moe-Ronickah** [comment]
@LordHelmchen98 , do not get snotty with any person who is helping on the GitHub.  Everyone here is volunteering their time and expertise, with family, school, and jobs also to consider.

**Comment by LordHelmchen98** [comment]
You surely misunderstood me I didnt want to harass anyone. I was just curious because there was no further reply. And you may have noticed I try to help others the best I can (also here on GitHub) and yes Im doing that in my freetime as well.

**Comment by Villageacules** [comment]
@LordHelmchen98 Can this be confirmed still a bug by you? Notify me somewhere if it is still happening.

**Comment by LordHelmchen98** [comment]
I'm not at home until Saturday, but as soon as I get home I'll check that.

**Comment by LordHelmchen98** [comment]
Yes @Villageacules I just checked it and this is still happening.
